exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -135,6 +135,8 @@ table.sortable thead .sorting_desc { background-image: url("/static/img/black-de
 
   <#assign queueName = model.getProfile().getQueueName() />
   <#assign queued = queueName != "" && queueName != "-" />
+  <#assign queryStateDisplayName = ["Starting", "Running", "Succeeded", "Canceled", "Failed",
```

Comment:
Why do we have need duplicated state display name array here and in `ProfileResources.java`. Is there a possibility to not duplicate it?

Reply:
Query state is displayed in two places - list.ftl (accessed via ProfileResources.java) and profile.ftl (accessed via QueryProfile.proto). So, I had to duplicate the array once in ProfileResources.java & once in profile.ftl

Reply:
Maybe you can create common freemarker function which list.ftl and profile.ftl will use to decode the state name thus we won't duplicate display names and always be sure they are in sync.

Reply:
Yes, I could create a common freemarker function. But if the changes are not made in ProfileResources.java then the REST API /profile will still show the query state as "COMPLETED". It won't be in sync with the UI.

Reply:
Prasad, maybe you can come up with different way to avoid display names duplication?